src/lib/commandline_cmds.ts

```diff
@@ -1,5 +1,17 @@
 import { messageOwnTab } from "@src/lib/messaging"
 import * as State from "@src/state"
+import { contentState } from "@src/content/state_content"
+
+const sleep = ms => new Promise(resolve => setTimeout(resolve, ms))
+
+async function awaitProxyEq(proxy, a: string, b: string) {
+    let counter = 0
+    while (proxy[a] != proxy[b] && counter < 50) {
+        await sleep(10)
+        counter += 1
+    }
+    return proxy[a] == proxy[b]
+}
```

Comment:
This seems extremely fragile to me. Don't we risk an enormous amount of breakage if two tabs run `ex.` commands at the same time? I can see the `counter` variable is there to prevent blocking forever but I'm still not sure this is a good idea.

Reply:
`contentState` is per-tab, so I don't think this can ever happen.